Add rule docs for 8.2 rule changes (#1863)
elastic/security-docs: docs/detections/prebuilt-rules/rule-details/potential-remote-credential-access-via-registry.asciidoc

[[potential-remote-credential-access-via-registry]]
=== Potential Remote Credential Access via Registry

Identifies remote access to the registry to potentially dump credential data from the SAM registry hive in preparation for credential access and privileges elevation.

*Rule type*: eql

*Rule indices*:

* winlogbeat-*
* logs-system.*
* logs-endpoint.events.*

*Severity*: high

*Risk score*: 73

*Runs every*: 5 minutes

*Searches indices from*: now-9m ({ref}/common-options.html#date-math[Date Math format], see also <<rule-schedule, `Additional look-back time`>>)

*Maximum alerts per execution*: 100

*References*:

* https://github.com/SecureAuthCorp/impacket/blob/master/examples/secretsdump.py

*Tags*:

* Elastic
* Host
* Windows
* Threat Detection
* Lateral Movement
* Credential Access

*Version*: 1

*Added ({stack} release)*: 8.2.0

*Rule authors*: Elastic

*Rule license*: Elastic License v2

==== Investigation guide

[source,markdown]
----------------------------------
## Triage and analysis

### Investigating Potential Remote Credential Access via Registry

Dumping registry hives is a common way to access credential information. Some hives store credential material, 

> such as the SAM hive, which stores locally cached credentials (SAM Secrets), and the SECURITY hive, which stores domain

Reviewer: ```suggestion
such as the SAM hive, which stores locally cached credentials (SAM secrets), and the SECURITY hive, which stores domain
```

cached credentials (LSA secrets). Dumping these hives in combination with the SYSTEM hive enables the attacker to
decrypt these secrets.

Attackers can use tools like secretsdump.py or CrackMapExec to dump the registry hives remotely, and use dumped
credentials to access other systems in the domain.

#### Possible investigation steps

- Identify the target host role, involved account, and source host.
- Determine the privileges assigned to any compromised accounts.
- Investigate other alerts related to the involved user and source host in the last 48 hours.

> - Scope potentially compromised accounts. Analysts can do this by searching for login events (e.g., 4624) to the target

Reviewer: ```suggestion
- Scope potentially compromised accounts. Analysts can do this by searching for login events (for example, 4624) to the target
```

host.

### False positive analysis

- False positives for this rule are unlikely. Any activity that triggered the alert and is not inherently malicious must
be monitored by the security team.

### Related rules

- Credential Acquisition via Registry Hive Dumping - a7e7bfa3-088e-4f13-b29e-3986e0e756b8

### Response and remediation

- Initiate the incident response process based on the outcome of the triage.
- Isolate the involved hosts to prevent further post-compromise behavior.
- Scope compromised credentials and disable the accounts.
- Reset the passwords of compromised accounts.
- Determine if other hosts were compromised.

## Config

> This rule uses Elastic Endpoint file creation and System Integration events for correlation. Both data should be

Reviewer: ```suggestion
This rule uses Elastic Endpoint file creation and system integration events for correlation. Both data should be
```
